nucypher/cli/characters/stake.py

```diff
@@ -225,12 +222,12 @@ def stake(click_config,
         emitter.echo(f"Successfully detached worker {worker_address} from staker {staking_address}", color='green')
         paint_receipt_summary(emitter=emitter,
                               receipt=receipt,
-                              chain_name=STAKEHOLDER.blockchain.client.chain_name,
+                              chain_name=blockchain.client.chain_name,
                               transaction_type='detach_worker')
         emitter.echo(f"Detached at period #{current_period} ({bonded_date})", color='green')
         return  # Exit
 
-    elif action == 'init':
+    elif action == 'create':
```

Comment:
Do we need to update the docs since `init` is now `create`?

Reply:
:heavy_check_mark: